src/test/java/bio/terra/workspace/integration/WorkspaceIntegrationTest.java

```diff
@@ -134,26 +131,6 @@ public void deleteWorkspace(TestInfo testInfo) throws Exception {
     testToWorkspaceIdsMap.remove(testInfo.getDisplayName());
   }
 
-  @Test
-  @Tag(TAG_NEEDS_CLEANUP)
-  public void deleteWorkspaceWithInvalidToken(TestInfo testInfo) throws Exception {
-    UUID workspaceId = UUID.randomUUID();
-    testToWorkspaceIdsMap.put(testInfo.getDisplayName(), Collections.singletonList(workspaceId));
-    WorkspaceResponse<CreatedWorkspace> workspaceResponse = createDefaultWorkspace(workspaceId);
-
-    String userEmail = testConfig.getUserEmail();
-    String token = "invalidToken";
-    String path = testConfig.getWsmWorkspacesBaseUrl() + "/" + workspaceId;
-    DeleteWorkspaceRequestBody body = new DeleteWorkspaceRequestBody().authToken(token);
-    String jsonBody = testUtils.mapToJson(body);
-
-    WorkspaceResponse<?> deleteWorkspaceResponse =
-        workspaceManagerTestClient.delete(userEmail, path, jsonBody);
-
-    assertEquals(HttpStatus.UNAUTHORIZED, deleteWorkspaceResponse.getStatusCode());
-    assertTrue(deleteWorkspaceResponse.isErrorObject());
-  }
-
```

Comment:
this test is no longer necessary. This API now only looks at the token in the HTTP headers ... and any invalid token in the header will be rejected by the proxy and not even make it to the application.